wls-gui-admintool/src/components/config-parameter/BaseCardConfigParameterDisplay.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <v-card
+    class="card"
+    :title="configParameter.name"
+    :subtitle="configParameter?.beschreibung"
+  >
+    <v-divider class="card-divider" />
+    <v-card-text class="card-content">
```

Comment:
Die CSS-Klasse gibt es nicht

```suggestion
    <v-card-text>
```